tests/images.rs

```diff
@@ -331,43 +331,33 @@ pub fn free_local_port() -> Option<u16> {
 }
 
 #[test]
-fn zookeeper_check_connection() {
-    struct LoggingWatcher;
-    impl Watcher for LoggingWatcher {
-        fn handle(&self, e: WatchedEvent) {
-            info!("{:?}", e)
-        }
-    }
-    use std::env;
-    use std::time::Duration;
-    let _ = pretty_env_logger::try_init();
-    let free_local_port = free_local_port().unwrap();
+fn zookeeper_check_existence_directories() {
+    let default_zookeeper_port = 2181;
+    let free_local_port = free_local_port().unwrap_or(default_zookeeper_port);
     let docker = clients::Cli::default();
-    let image = images::zookeeper::Zookeeper::default().with_mapped_port((free_local_port, 2181));
+    let image = images::zookeeper::Zookeeper::default()
+        .with_mapped_port((free_local_port, default_zookeeper_port));
```

Comment:
You don't need to specify this again here, just calling `node.get_host_port` should be sufficient!